_topic_maps/_topic_map.yml

```diff
@@ -2569,8 +2569,13 @@ Distros: openshift-origin,openshift-enterprise
 Topics:
 - Name: Red Hat OpenShift support for Windows Containers overview
   File: index
-- Name: Red Hat OpenShift support for Windows Containers release notes
-  File: windows-containers-release-notes-10-15-x
+- Name: Release notes
+  Dir: windows_rn
+  Topics:
+  - Name: Red Hat OpenShift support for Windows Containers release notes
+    File: windows-containers-release-notes-10-15-x
+  - Name: Past releases
+    File: windows-containers-release-notes-10-15-x-past
```

Comment:
I was kind of thinking about the structure of the subdirectories here and possibly some of the naming.
The most current release notes seem to get buried under the support information. So what do you think of making support its own assembly (I know it has 10.15.z info in it though) and the current release notes its own assembly? It might be too much to maintain though is my only concern. 

or you could just rename the assembly to "Red Hat OpenShift for Windows Containers release notes" since the first section is support.